src/middlewared/middlewared/alert/source/cores.py

```diff
@@ -16,18 +22,39 @@ class CoreFilesArePresentAlertClass(AlertClass):
 
 
 class CoreFilesArePresentAlertSource(AlertSource):
+    dumps = {}
+    dump_error_logged = []
     schedule = IntervalSchedule(timedelta(minutes=5))
 
     products = ("SCALE",)
 
     async def check(self):
         corefiles = []
-        for coredump in await self.middleware.call("system.coredumps"):
-            if coredump["corefile"] == "present":
-                if coredump["exe"] == "/usr/sbin/syslog-ng":
+        for coredump in filter(lambda c: c["corefile"] == "present", await self.middleware.call("system.coredumps")):
+            if coredump["exe"] == "/usr/sbin/syslog-ng":
+                continue
+
+            if coredump["pid"] not in self.dumps:
+                cp = await run(
```

Comment:
We need to skip running `coredumpctl info` if `coredump["pid"]` is in `self.dump_error_logged`, otherwise we will be running it every minute for each malformed coredump.